docker-storage-setup.sh

```diff
@@ -1032,6 +1008,19 @@ check_storage_options(){
   if [ "$DOCKER_ROOT_VOLUME" == "yes" ];then
      Info "DOCKER_ROOT_VOLUME is deprecated, and will be removed soon. Use CONTAINER_ROOT_LV_MOUNT_PATH instead."
   fi
+
+  if [ ! -z "$DOCKER_ROOT_VOLUME_SIZE" ]; then
+      Info " DOCKER_ROOT_VOLUME_SIZE is deprecated, please use CONTAINER_ROOT_LV_SIZE"
+  fi
+
```

Comment:
Why not use `-n` instead of ` ! -z` ? That would be more inline with the existing checks and less characters.